srcpkgs/iptables/template

```diff
@@ -12,14 +12,7 @@ maintainer="Anthony Iliopoulos <ailiop@altatus.com>"
 license="GPL-2.0-or-later"
 homepage="https://www.netfilter.org/"
 distfiles="https://www.netfilter.org/projects/iptables/files/iptables-${version}.tar.bz2"
-checksum=d457d74512e63aa3f50336e0597d4023c0e3c6845594d38532efb6ebcb294309
-	alternatives="
-		iptables:ip6tables:/usr/bin/xtables-legacy-multi
-		iptables:ip6tables-restore:/usr/bin/xtables-legacy-multi
-		iptables:ip6tables-save:/usr/bin/xtables-legacy-multi
-		iptables:iptables:/usr/bin/xtables-legacy-multi
-		iptables:iptables-restore:/usr/bin/xtables-legacy-multi
-		iptables:iptables-save:/usr/bin/xtables-legacy-multi"
```

Comment:
Does it create the symlinks by itself, then?

Reply:
The hell happened there